front/pages/w/[wId]/assistant/[cId]/index.tsx

```diff
@@ -83,6 +84,9 @@ export default function AssistantConversation({
       window.alert(`Error creating message: ${data.error.message}`);
       return;
     }
+    await mutate(
+      `/api/w/${owner.sId}/assistant/conversations/${conversationId}/messages`
```

Comment:
Do we have such endpoint?

also next recommend using useSwrConfig to get the mutate and not a direct import (see my PR yesterday)